### ISSUE TITLE
Flask and Zappa requires different Werkzeug version 

### ISSUE THREAD

**Issue by kazitanvirahsan** [zappa/Zappa#1003]
Hai Please look at the below screenshots. Do I have to install an older version of Flask to make it work with Zappa? Can anyone help me?
<img width="1013" alt="Screen Shot 2021-07-09 at 9 42 59 pm" src="https://user-images.githubusercontent.com/6487015/125073187-eef0b700-e0fe-11eb-9ae7-fe8e7468c1e0.png">

**Comment by pedro-hdt** [comment]
I found that it is fine to use Zappa in your general user installation and just have Flask inside your virtual environment

**Comment by Ashatz** [comment]
I noticed this same issue as well.  I first installed Flask (2.0.1) and when I installed zappa (0.53.0), it retrograded my Werkzeug version down to 0.16.  

I was able to get around this by uninstalling and re-installing Flask.  I was then able to successfully run a simple deployment.  (_zappa deploy/update stuff_).

What exactly is zappa using this dependency for, and would it be possible to update to the latest and greatest?  

Thank you for your time.
